recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - lz4
     - numpy >=1.18
     - pandas >=1.0
-    - bokeh >=2.4.2
+    - bokeh >=2.4.2,<3
```

Comment:
We may want to add a repodata hot-fix to constrain older versions similarly

Reply:
Can pick this up if no one else has looked into it yet